src/store/modules/profile.js

```diff
@@ -0,0 +1,22 @@
+import { fetchUsers } from '@/api/users'
```

Comment:
You can merge it with Users module

Reply:
I was thinking about this but wasn't sure. Merged it with user module, and removed the profile module